src/app/kbv/controllers/single-amount-question.js

```diff
@@ -51,9 +52,16 @@ class SingleAmountQuestionController extends BaseController {
       if (err) {
         return callback(err);
       }
-
-      const userInput = req.body[req.session.question.questionKey];
-
+  
+      let userInput = req.body[req.session.question.questionKey];
+  
+      switch (req.session.question.questionKey) {
+        case 'rti-p60-earnings-above-pt':
+          userInput = fields.stripSpaces(userInput);
```

Comment:
We can probably move this up to line 57 as I think really we want to strip spaces for all inputs?